docs/usage/configuration/config_documentation.md

```diff
@@ -2837,6 +2837,17 @@ Example configuration:
 ```yaml
 track_appservice_user_ips: true
 ```
+---
+### `use_appservice_legacy_authorization`
+
+Whether to send the application service access token via the `access_token` query parameter.
+Defaults to false. This option is considered insecure and is not recommended. 
```

Comment:
I think we like to be explicit about what setting it to the non-default value does? I also offered a bit more info, what do you think?

```suggestion
Whether to send the application service access tokens via the `access_token` query parameter
per older versions of the Matrix specification. Defaults to false. Set to true to enable sending
access tokens via a query parameter.

**Enabling this option is considered insecure and is not recommended. **
```

Reply:
Sounds good to me, thanks for suggesting!